cluster/examples/kubernetes/ceph/object-multisite.yaml

```diff
@@ -0,0 +1,30 @@
+#################################################################################################################
+# Create an object store with settings for a test environment. Only a single OSD is required in this example.
+#  kubectl create -f object-test.yaml
+#################################################################################################################
+
+apiVersion: ceph.rook.io/v1
+kind: CephObjectStore
+metadata:
+  name: multisite-store
+  namespace: rook-ceph
+spec:
+  metadataPool:
+    failureDomain: host
+    replicated:
+      size: 3
+      requireSafeReplicaSize: true
+  dataPool:
+    failureDomain: host
+    replicated:
+      size: 3
+      requireSafeReplicaSize: true
+    compressionMode: none
+  preservePoolsOnDelete: false
+  gateway:
+    type: s3
+    port: 80
+    securePort:
+    instances: 1
+  multisite:
```

Comment:
If the only difference between the object store in multisite.yaml and object.yaml is the zone property, we should consider just adding this to object.yaml, but comment it out by default. This will could help reduce the yaml duplication.